public/assets/css/main.css

```diff
@@ -1,2 +1,2 @@
-@import url("font-awesome.min.css");
```

Comment:
You do want the font-awesome CSS since that allows the cool, quick loading, icons for navigation and social media. You don't need the main.css file to do this. You just need to link the the font-awesome css file from your index page.